sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/jit/ToJitInnerVisitor.java

```diff
@@ -360,11 +360,16 @@ public JITBlock getCurrentBlock() {
      * @param name           Name of function to call.
      * @param expression     Operations which is being translated.
      *                       Usually an ApplyExpression, but not always.
+     * @param resultType     Expected result type.  If null, type obtained from expression.
```

Comment:
parameter order in the docs here should match the actual parameter order

Reply:
I reordered them, but, frankly, I don't see the point of this. The IDEs use the definitions no matter what the order.
So there is no real benefit having the same order - just more work for me.